zmon-common/src/main/java/org/zalando/zmon/exception/CheckDefinitionNotFoundException.java

```diff
@@ -0,0 +1,22 @@
+package org.zalando.zmon.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@ResponseStatus(value=HttpStatus.NOT_FOUND, reason = "Check ID not found. Try again with a valid check Id!")
+public class CheckDefinitionNotFoundException extends ZMonException{
+
+    public static final ZMonExceptionFactory FACTORY = new ZMonExceptionFactory() {
```

Comment:
You don't seem to be using these factories yet. Do we really need them?

Reply:
I don't think it is used so updated it.